bokeh/models/renderers.py

```diff
@@ -51,6 +51,27 @@ class TileRenderer(Renderer):
     level = Enum(RenderLevel, default="underlay", help="""
     Specifies the level in which to render the glyph.
     """)
+    render_parents = Bool(default=True, help="""
+    Flag enable/disable drawing of parent tiles while waiting for new tiles to arrive. Default value is True.
+    """)
+
+class DynamicImageRenderer(Renderer):
+
+    image_source = Instance(ImageSource, help="""
+    Image source to use when rendering on the plot.
+    """)
+
+    alpha = Float(1.0, help="""
+    tile opacity 0.0 - 1.0
+    """)
+
+    level = Enum(RenderLevel, default="underlay", help="""
```

Comment:
Yes, so the `render_parents` flags is used in the scenario that you have a tile layer which is purely labels, its visually messy if parent tiles are rendered in-place of tiles currently being fetched.  This came into play during datashader.